server/sonar-server/src/main/java/org/sonar/server/issue/ws/SearchAction.java

```diff
@@ -432,7 +424,10 @@ protected void writeFacets(Request request, SearchOptions options, SearchResult<
           json.prop("count", bucket.getValue());
           json.endObject();
         }
-        addZeroFacetsForSelectedItems(request, facetName, itemsFromFacets, json);
+        // Prevent appearance of a glitch value due to dedicated parameter for this facet
```

Comment:
``addZeroFacetsForSelectedItems`` adds zero-valued items to a facet using parameters from the request. E.g, when you put ``assignees=julien,simon``, and when ``julien`` has never had any issue assigned to him, its login will not appar in the ES aggregation results, so we manually add a value ``julien: 0``.

This can't work with the ``assigned_to_me`` facet, because it is not associated to a search parameter.